contracts/utils/ChainTracker.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IChainTracker } from '../interfaces/IChainTracker.sol';
+
+abstract contract ChainTracker is IChainTracker {
+    // uint256(keccak256('chain_tracker_slot')) - 1
```

Comment:
use the same derivation as other contracts?
https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/gateway/BaseAmplifierGateway.sol#L11